bundler/spec/update/gems/fund_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+RSpec.describe "bundle update" do
+  let(:config) {}
+
+  before do
+    gemfile <<-G
+      source "#{file_uri_for(gem_repo1)}"
+      gem 'has_metadata'
+      gem 'has_funding', '< 2.0'
+    G
+
+    bundle "config set #{config}" if config
+
+    bundle :install
+  end
+
+  shared_examples "a fund message outputter" do
+    it "should display fund message for updated gems" do
+      expect(out).to include("2 gems you directly depend on are looking for funding.")
+    end
+  end
```

Comment:
It seems like it's better to move this shared spec inline since there's only one spec using it?

Reply:
So, this will repeat the same message every time `bundle update` is run, just like if `bundle install` had been run, right? I was thinking if this should only inform about funding of gems that were explicitly updated. But it sounds tricky and potentially misleading. This seems fine.